coalib/settings/Setting.py

```diff
@@ -133,12 +160,30 @@ def __path__(self, origin=None):
 
         if hasattr(self, "origin") and self.origin != "":
             origin = self.origin
+        if glob_escape_origin:
+            origin = glob_escape(origin)
 
         if origin is None:
             raise ValueError("Cannot determine path without origin.")
 
         return os.path.abspath(os.path.join(os.path.dirname(origin), strrep))
```

Comment:
True. Don't know how I missed it. (BTW I think an abspath before Line 163 on the right is enough.)